tests/DifferTest.php

```diff
@@ -475,19 +475,21 @@ public function textForNoNonDiffLinesProvider(): array
         ];
     }
 
+    /**
+     * @expectedException \InvalidArgumentException
+     * @expectedExceptionMessageRegExp #^"from" must be an array or string\.$#
+     */
     public function testDiffToArrayInvalidFromType()
     {
-        $this->expectException('\InvalidArgumentException');
-        $this->expectExceptionMessageRegExp('#^"from" must be an array or string\.$#');
-
         $this->differ->diffToArray(null, '');
     }
 
+    /**
+     * @expectedException \InvalidArgumentException
+     * @expectedExceptionMessageRegExp #^"to" must be an array or string\.$#
+     */
     public function testDiffInvalidToType()
     {
-        $this->expectException('\InvalidArgumentException');
-        $this->expectExceptionMessageRegExp('#^"to" must be an array or string\.$#');
```

Comment:
ah damn, I started out with support for 5.5.9 when I didn't release we only need 5.6, will revert :+1: